interactive_demo.py

```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/env python
+
+import math
+import os
+
+from drake_ros.systems import MoveableJoints
+from drake_ros.systems import MoveablePoint
+from drake_ros.systems import SystemClock
+from drake_ros.systems import TFPublisher
+
+from interactive_markers import InteractiveMarkerServer
+
+import numpy
+
+from pydrake.common.eigen_geometry import Quaternion
+from pydrake.common.value import AbstractValue
+from pydrake.examples.manipulation_station import ManipulationStation
+from pydrake.geometry import ConnectDrakeVisualizer
+from pydrake.manipulation.planner import DifferentialInverseKinematicsParameters
```

Comment:
Can you format these according to this `isort` settings file?
https://github.com/RobotLocomotion/drake/pull/13709
https://github.com/RobotLocomotion/drake/blob/985cf2167fb5c388614137c65796772ad3e98f6d/pyproject.toml

Reply:
isort'd in 95fb9c1